www/includes/easyparliament/hansardlist.php

```diff
@@ -2309,7 +2319,16 @@ public function _get_data_by_gid($args) {
 
         // We need to get the data for this gid.
         // Then depending on what htype it is, we get the data for other items too.
-        global $DATA, $this_page, $hansardmajors;
+        global $DATA, $this_page, $hansardmajors, $MEMBER, $THEUSER;
+
+        // use this for generating member vote data
+        if (isset($THEUSER) && $THEUSER->postcode_is_set()) {
```

Comment:
I wonder if this should instead go in the www/docs/section.php file, so it's set up outside this class (which shouldn't rreally need to know about it here).